modules/openapi-generator/src/main/java/org/openapitools/codegen/OpenAPINormalizer.java

```diff
@@ -912,7 +912,7 @@ public boolean isNullTypeSchema(Schema schema) {
         }
 
         if (schema instanceof JsonSchema) { // 3.1 spec
-            if (Boolean.TRUE.equals(schema.getNullable())) {
+            if (Boolean.TRUE.equals(schema.getNullable()) && schema.get$ref() == null) {
```

Comment:
Could you please add to [bin/configs/](https://github.com/OpenAPITools/openapi-generator/tree/master/bin/configs) a small example of a case where this change prevents the generation of invalid code?
Thanks.

Reply:
I tried long and hard, but couldn't reproduce this within the sample specs. Can you take a look? Just take [this spec](https://raw.githubusercontent.com/github/rest-api-description/main/descriptions-next/api.github.com/api.github.com.json) and you will see the error in e.g. `model.RepositoryWebhooks`, where you will see `AnyOf license` instead of `SimpleLicense license`.